pkg/api/volume/handler.go

```diff
@@ -95,32 +96,61 @@ func (h *ActionHandler) Do(ctx *harvesterServer.Ctx) (interface{}, error) {
 	}
 }
 
-func (h *ActionHandler) validateExportVolume(storageClassName, pvcNamespace, pvcName string) string {
+func (h *ActionHandler) checkPVCInUsed(pvcNamespace, pvcName string) error {
```

Comment:
would it make sense to conver the logic to an indexer that indexes pods by pvc names and then we can just query pod status if pvc is attached to any pod